### DISCUSSION TITLE
How to set focus after new node

### DISCUSSION THREAD

**Discussion by jiphelsen** [JiHong88/SunEditor#1074] [Q&A] [answered]
Hi I was trying to make a command plugin to add in page breaks. Here is my code for the action :

action: function () {
        if (!this.util.hasClass(this.context.addPageBreak.targetButton, 'active')) {
            const newNode = this.util.createElement('DIV')
            newNode.style = 'display: block;  page-break-after: always;'
            this.nodeChange(newNode, ['display','page-break-after'], null, null);
        }
    }

I however noticed that nodeChange results in the cursor being set inside the new div element. 
Is there anyway to set the focus always after the newly created element?
Thanks in advance, jip

**Comment by JiHong88** [comment, marked as answer]
Try it.
this.nodeChange(newNode, ['display','page-break-after'], null, null);
`this.setRange(newNode, 1, newNode, 1)`

**Reply by jiphelsen** [reply]
Thank you! Works like a charm. Big fan of SunEditor!
